src/routes/beta/lib/editor/VisualEditor2.svelte

```diff
@@ -586,6 +586,13 @@
         the sockets in place.
       </p>
     </InfoBox>
+  {:else if $protoConfig.partType.type == 'niz'}
+    <InfoBox>
+      <p>
+        This variant requires direct pin assignment to the microcontroller and a well-tuned 3D printer.
```

Comment:
I assume this is because the Niz switch is press-fitted into the socket? It'd be great if we had a solution that doesn't require press-fitting, but I see little helpful to create a latching mechanism in the Niz switch housing.

P.S. The PR looks A-OK to me!

Reply:
Correct, they are press fit.
They do have slots for tabs, but IMO, it would be tough to 3d print something with FFM. Less sure about resin.

I may eventually look for a different HE sensor again. Getting it into a matrix setup would definitely be ideal.

Reply:
Ah, I see. Tabs are pretty tough to print consistently across FDM printers.

Thanks again for the PR! It would be cool to see this working with a matrix, but I'm out of ideas on how to make it work.